archive/demos/shiny-drum-machine-audio.js

```diff
@@ -0,0 +1,307 @@
+/* eslint require-jsdoc: "off" */
+
+import {INSTRUMENTS, freeze, clone} from './shiny-drum-machine-data.js';
+
+const AudioContext = window.AudioContext ?? window.webkitAudioContext;
```

Comment:
I have not tested, but do we still need this?

Reply:
Safari 14.1.1 and Firefox 89 support it, removing the vendor-prefixed version.